apps/meeting/serverless/src/handlers.js

```diff
@@ -110,21 +124,32 @@ exports.createMeeting = async (event, context, callback) => {
   }
 
   let meetingInfo = await getMeeting(title);
+  let client = getClientForMeeting(meetingInfo);
   if (!meetingInfo) {
-    const request = {
+    let request = {
       ClientRequestToken: uuid(),
       MediaRegion: region,
       NotificationsConfiguration: getNotificationsConfig(),
+      ExternalMeetingId: title.substring(0, 64),
     };
+    if (event.queryStringParameters.ns_es === 'true') {
+      client = chimeRegional;
+      request.MeetingFeatures = {
+        Audio: {
+          // The EchoReduction parameter helps the user enable and use Amazon Echo Reduction.
```

Comment:
Could we make it two lines to reduce the line width?
And this comment seems to be too general, could we explain a bit more like how it helps?

Reply:
i see it is 100 characters so should be fine. this is also keeping the same from JS SDK.
https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/server.js#L94